src/js/components/DataTableGroupBy/propTypes.js

```diff
@@ -0,0 +1,5 @@
+let PropType = {};
+if (process.env.NODE_ENV !== 'production') {
+  PropType = {};
```

Comment:
Looks like we are missing `options` here

Reply:
added propTypes